content/millennium/r4/financial/coverage.md

```diff
@@ -89,6 +89,14 @@ Search for Patient-level or Encounter-level Coverages that meet supplied query p
 <%= headers status: 200 %>
 <%= json(:r4_coverage_encounter_bundle) %>
 
+
+#### Response when social healthcare information is returned
```

Comment:
are we calling this social or public healthcare externally?

Reply:
Updated [here](https://github.com/cerner/fhir.cerner.com/commit/bfdeb4b16e342fb06798c33c70555aec4b30329e#diff-81b4004b2ae7fd9a288412597630810cL93)